### PR TITLE
feat: build and publish skopeo and zstd

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

Launcher docker builds are broken as it downloads pkgs from bintray service which has been sunset

## Objective

This PR builds, pkgs and publishes skopeo and zstd binaries

## References

https://github.com/screwdriver-cd/screwdriver/issues/2450

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: screwdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+jobs:
+  skopeo-build:
+    requires: [~pr, ~commit]
+    image: centos:centos7
+    steps:
+      - make: |
+          yum install -y epel-release
+          yum install -y wget git make gcc gcc-c++ jq
+          wget -q -O go1.15.6.tar.gz https://golang.org/dl/go1.15.6.linux-amd64.tar.gz
```
Comment (adong) [inline screwdriver.yaml:16]: `1.15.6` is hard-coded here.

File: screwdriver.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  zstd-build:
+    requires: [~pr, ~commit]
+    image: centos:centos7
+    steps:
+      - make: |
+          yum install -y epel-release
+          yum install -y zlib-devel wget make gcc gcc-c++ jq
+          wget -q -O zstd-1.5.0.tar.gz https://github.com/facebook/zstd/releases/download/v1.5.0/zstd-1.5.0.tar.gz
+          tar -C . -oxzf zstd-1.5.0.tar.gz
```
Comment (adong) [inline screwdriver.yaml]: So is `zstd`'s version.

Can we export the zstd version as environment variables like `ZSTD_PACKAGE` ?

File: screwdriver.yaml (hunk 1 above)
Comment (pritamstyz4ever) [reply screwdriver.yaml:16]: yeah, that's the version we use in launcher

File: screwdriver.yaml (hunk 2 above)
Comment (pritamstyz4ever) [reply screwdriver.yaml]: These are the binaries we build to be used in the launcher and are tied to their specific version. If we update a version it gets updated in both places. Not sure if that helps anything

**Review by tkyi — DISMISSED** [review]

File: screwdriver.yaml (hunk 2 above)
Comment (adong) [reply screwdriver.yaml]: I meant merely this

```

shared:
  image: node:12
  environment:
    SKOPEO_PACKAGE: skopeo
    ZSTD_PACKAGE: zstd
    ZSTD_PACKAGE_VERSION: 1.5.0

```


```

wget -q -O zstd-${ZSTD_PACKAGE_VERSION}.tar.gz https://github.com/facebook/zstd/releases/download/v${ZSTD_PACKAGE_VERSION}/zstd-${ZSTD_PACKAGE_VERSION}.tar.gz
  tar -C . -oxzf zstd-${ZSTD_PACKAGE_VERSION}.tar.gz
```

**Review by tkyi — APPROVED** [review]

**Pull request merged**